system/file_test.go

```diff
@@ -68,7 +67,8 @@ func TestCopyFile(t *testing.T) {
 }
 
 func TestListDir(t *testing.T) {
-	tmpDir, err := ioutil.TempDir("", "test")
+	tmpDirRaw, err := os.CreateTemp("", "test")
+	tmpDir := tmpDirRaw.Name()
```

Comment:
As we need a temporary **directory** and not a file, may be `os.MkdirTemp` will be the better choice?

Reply:
Yes, you are right!